packages/components/select/src/useSelect.ts

```diff
@@ -210,6 +210,15 @@ export const useSelect = (props, states: States, ctx) => {
     () => props.placeholder,
     (val) => {
       states.cachedPlaceHolder = states.currentPlaceholder = val
+
+      const hasValue =
+        props.multiple &&
+        Array.isArray(props.modelValue) &&
+        props.modelValue.length > 0
```

Comment:
## Suggest

A duplicate of that code exists at line `114`.

I think it would be better to separate it into computed and reuse it.